.github/workflows/ci.yaml

```
@@ -234,6 +234,25 @@ jobs:
       - uses: r7kamura/rust-problem-matchers@v1
       - run: cargo +${{ matrix.rust }} test
 
+  test-wasm:
+    runs-on: ubuntu-22.04
+    needs:
+      - lint
+    strategy:
+      matrix:
+        rust:
+          - stable
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ matrix.rust }}
+      - name: Install wasm-pack
+        run: |
+          curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Sorry to double up on this, but we're looking to start using `taiki-e/install-action` wherever possible:
```suggestion
      - uses: taiki-e/install-action@v2
        with:
          tool: wasm-pack
```

---

thanks

---

i just copied how `mobilecoin.git` is working, this looks find too though